db/migrate/1_create_pay_tables.rb

```diff
@@ -11,7 +11,7 @@ def change
       t.datetime :deleted_at
       t.timestamps
     end
-    add_index :pay_customers, [:owner_type, :owner_id, :deleted_at, :default], name: :pay_customer_owner_index
```

Comment:
Now that I'm thinking about it, `:deleted_at` might be the safer one to keep. `:default` is a boolean and in theory a faster query?